Shopify/unity-buy-sdk
# Respond address changes to Android Pay

### Task description:

Whenever the user updates their shipping information from the plugin, we'll need to pass the new information back to Unity to update the Checkout object. We'll then need to take the new Checkout object from GraphQL and pass the information back to Android to update our Pay Cart.

#### Task bullet:

Handle Checkout response in Unity and pass updated information back to Android to update the PayCart

### How to test:

If you want to see it on your phone, make sure Android Pay is in sandbox mode.

Then, run:

```bash
cd <unity-buy-sdk-examples-dir>
curl -L 'https://github.com/Shopify/unity-buy-sdk/files/1360866/NativePay.txt' > /tmp/NativePay.txt; patch -p1 < /tmp/NativePay.txt
```

This will change the example project to use native pay instead of web pay.

Then, generate the unity package, import it in Unity tool and follow all the steps to run it on the phone.

### PR expectations:

The code is still dirty and this PR is not meant to be merged, this is just a request for get early feedback. No tests have been written yet.
